docs/UsingLayering.md

```diff
@@ -0,0 +1,165 @@
+# Using OCP CoreOS Layering Phase 0
```

Comment:
I know this is just to point people at, but I think it's worth putting a link in our readme to this doc, since it is pretty important!

Reply:
I added a line in the README after the part where we talk about machineconfig/ignition. It seemed like a good place but I didn't feature it super-prominently. Let me know if you'd rather it go somewhere else?